package-lock.json

```diff
@@ -14315,7 +14269,6 @@
                 "escape-string-regexp": "4.0.0",
                 "find-up": "5.0.0",
                 "glob": "7.2.0",
-                "he": "1.2.0",
```

Comment:
mocha needs this. I'm guessing this was a "manual" edit. Looks like mocha has a dependency on "he". But mocha is a dev-dep and that doesn't change the fact that we should remove our direct dep on "he".